docs/detections/rules-ui-create.asciidoc

```diff
@@ -29,13 +29,27 @@ generated for each value.
 For example, if the threshold `field` is `source.ip` and its `value` is `10`, an
 alert is generated for every source IP address that appears in at least 10 of
 the rule's search results.
+* *EQL*: Searches the defined indices and creates an alert when results match an
```

Comment:
In the UI we decided to name this rule type "Event Correlation". I wonder if we should do that in this section also. @paulewing thoughts?